data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/vnode-vault-creation-v1.0.xml

```diff
@@ -72,6 +72,19 @@
         <addPrimaryKey columnNames="group_id, member_name"
                        constraintName="vnode_member_info_pkey"
                        tableName="vnode_member_info"/>
+
+        <createTable tableName="vnode_group_policy">
+            <column name="schema_version" type="INT" autoIncrement="true">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="effective_from" type="TIMESTAMP">
```

Comment:
We could have many group policies effective at one time (depending on what CPI version / group policy version a member has installed). Maybe `created` would be a better name for this column?